.github/workflows/openvex.yml

```
@@ -0,0 +1,20 @@
+name: openvex
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - 'v*.*.*'
+jobs:
+  vexctl:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Set environment variables
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - uses: openvex/generate-vex@c59881b41451d7ccba5c3b74cd195382b8971fcd
+        # Refer: https://github.com/openvex/vexctl#operational-model
+        name: Run vexctl
+        with:
+            product: pkg:golang/kubernetes/kube-state-metrics@${{ env.RELEASE_VERSION }}
```

```suggestion
            product: pkg:golang/k8s.io/kube-state-metrics/v2@${{ env.RELEASE_VERSION }}
```

should this be the golang package name?

---

See also: https://github.com/package-url/purl-spec/blob/master/PURL-TYPES.rst#golang

---

Why the v2? Is it not covered by the env?

---

v2 is because it expects the full name of the golang package.